backend/api/Services/AcquisitionFileService.cs

```diff
@@ -0,0 +1,73 @@
+using System.Security.Claims;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Pims.Core.Extensions;
+using Pims.Dal.Entities;
+using Pims.Dal.Helpers.Extensions;
+using Pims.Dal.Repositories;
+using Pims.Dal.Security;
+
+namespace Pims.Api.Services
+{
+    public class AcquisitionFileService : IAcquisitionFileService
+    {
+        private readonly ClaimsPrincipal _user;
+        private readonly ILogger _logger;
+        private readonly IAcquisitionFileRepository _acqFileRepository;
+
+        public AcquisitionFileService(
+            ClaimsPrincipal user,
+            ILogger<AcquisitionFileService> logger,
+            IAcquisitionFileRepository acqFileRepository)
+        {
+            _user = user;
+            _logger = logger;
+            _acqFileRepository = acqFileRepository;
+        }
+
+        public PimsAcquisitionFile GetById(long id)
+        {
+            _logger.LogInformation("Getting acquisition file with id {id}", id);
+            _user.ThrowIfNotAuthorized(Permissions.AcquisitionFileView);
+
+            var acqFile = _acqFileRepository.GetById(id);
+            return acqFile;
+        }
+
+        public PimsAcquisitionFile Add(PimsAcquisitionFile acquisitionFile)
+        {
+            acquisitionFile.ThrowIfNull(nameof(acquisitionFile));
+
+            _logger.LogInformation("Adding acquisition file with id {id}", acquisitionFile.Id);
+            _user.ThrowIfNotAuthorized(Permissions.AcquisitionFileAdd);
+
+            acquisitionFile.AcquisitionFileStatusTypeCode = "ACTIVE";
+
+            var newAcqFile = _acqFileRepository.Add(acquisitionFile);
+            _acqFileRepository.CommitTransaction();
+            return newAcqFile;
+        }
+
+        public PimsAcquisitionFile Update(PimsAcquisitionFile acquisitionFile)
+        {
+            acquisitionFile.ThrowIfNull(nameof(acquisitionFile));
+
+            _logger.LogInformation("Updating acquisition file with id {id}", acquisitionFile.Id);
+            _user.ThrowIfNotAuthorized(Permissions.AcquisitionFileEdit);
+
+            ValidateVersion(acquisitionFile.Id, acquisitionFile.ConcurrencyControlNumber);
+
+            // TODO: Implementation pending
+            throw new System.NotImplementedException();
```

Comment:
This is a placeholder implementation for now that throws an exception - but includes boilerplate for whenever this story gets implemented